trip_app/trips/tests/test_views.py

```diff
@@ -42,66 +46,159 @@ def trips_with_different_coordinates():
-    assert resp.status_code == 201
-    trip_dict = resp.json()
-    assert trip_dict["driver"] == "admin"
-    assert trip_dict["dep_time"] == data["dep_time"]
-    assert trip_dict["start_point"] == data["start_point"]
-    assert trip_dict["dest_point"] == data["dest_point"]
-    assert trip_dict["price"] == data["price"]
-    assert trip_dict["num_seats"] == data["num_seats"]
-    assert trip_dict["description"] == data["description"]
-
-
-@pytest.mark.django_db
-def test_get_trips_filtering_time(admin_client, trips):
-    """ Verify that correct number of trips is created and returned queryset
-        is limited to the page_size parameter or TripListCreateApiView """
-    trips_count = len(trips)
-    resp = admin_client.get(list_create_trips_url)
-    assert resp.status_code == 200
-    trip_dict = resp.json()
-    assert trip_dict["count"] == trips_count
-    max_page_size = TripListCreateApiView.pagination_class.page_size
-
-    if trips_count > max_page_size:
-        assert len(trip_dict["results"]) == max_page_size
-    else:
-        assert len(trip_dict["results"]) == trips_count
-
-
-@pytest.mark.django_db
-def test_filter_trips_by_distance(admin_client, trips_with_different_coordinates):
-    """ Verify that trips are filtered correctly by minimum distance between user coordinates and trips coordinates """
-    exp_trip_order = [trip.id for trip in trips_with_different_coordinates]
-    user_sp, user_dp = trips_with_different_coordinates[0].start_point, trips_with_different_coordinates[0].dest_point
-    resp = admin_client.get(f"{list_create_trips_url}?sp={user_sp.x},{user_sp.y}&dp={user_dp.x},{user_dp.y}")
-    assert resp.status_code == 200
-    resp_dict = resp.json()
-    assert resp_dict['count'] == 5
-    results_dict = resp_dict['results']
-    act_trip_order = [trip["id"] for trip in results_dict]
-    assert act_trip_order == exp_trip_order
-    # As user start and destination points are the same as for trip 1
-    # the dist1 and dist2 parameters should be 0
-    assert results_dict[0]["dist1"] == '0.0 m'
-    assert results_dict[0]["dist2"] == '0.0 m'
-
-
-@pytest.mark.django_db
-def test_filter_trips_by_time(admin_client, trips_with_different_dep_times):
-    """ Verify that trips are filtered correctly by time and
-        trips that are between time1 and time2 interval are returned """
-    too_late_trip, in_time_trip = trips_with_different_dep_times
-    now = timezone.now()
-    # Time points between which we are looking for a trips
-    time_1_str = now.strftime("%Y-%m-%dT%H:%M:%S")
-    time_2_str = (now + timedelta(minutes=20)).strftime("%Y-%m-%dT%H:%M:%S")
-    resp = admin_client.get(f"{list_create_trips_url}?time1={time_1_str}&time2={time_2_str}")
-    assert resp.status_code == 200
-    resp_dict = resp.json()
-    assert resp_dict["count"] == 1
-    assert resp_dict["results"][0]["id"] == in_time_trip.id
+class TestTrips:
+    def test_create_trip(self, admin_client, trip_data):
```

Comment:
I'd say that `trip` is more or less redundant in the method names (as `TestTrips` already suggests that)

Reply:
I would agree, in this case, though there might be cases where being more explicit might be justified,
e.g. for GET it is common to have endpoints for getting list and individual entry. In this case I think including trip in function name `test_get_trip` should be appreciated even is there is GET /trips/ endpoint. And it brings up the question: should I be consistent across the test names in the same file which means to include 'trip' part of names everywhere or to include it only where it might be perceived ambiguous ? I haven't got strong opinion on this.....

Reply:
I'd follow the approach:
1. if namespace (class name) defines clearly what we are working on, shorten the method names 
1. do specify extra name suffix in more specific cases
Anyway, it makes sense to keep reasonably short method names wherever possible.

Reply:
Thanks @andreybulygin. I changed names of the tests in this file according to this approach.